### PR TITLE
Adding files to pytest example project

### PR DESCRIPTION
**Pull request by ankit-dq**

### CODE REVIEW

**Review by jthickens — DISMISSED** [review]

File: Python/pytest/resources/.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Download the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) and place it in this directory.
```
Comment (jthickens) [inline Python/pytest/resources/.txt:1]:
```suggestion
Download the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) for your operating system and place it in this directory.
```

File: Python/pytest/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+![logo](./docs/logo-python-pytest.png)
+
+# axe DevTools Python pytest API Example
+
+Using axe DevTools Python pytest, you can integrate axe DevTools into your existing testing environment. This example project demonstrates how axe DevTools Python pytest is used to detect accessibility issues and generates reports in CSV, HTML, JSON, and XML formats.
+
+## Prerequisites
+
+Python 3 (does not support Python 2) along with the virtual environment (venv or pyenv)
```
Comment (jthickens) [inline Python/pytest/README.md:9]:
```suggestion
Python 3 (does not support Python 2) along with the virtual environment (venv or pyenv).
```

**Review by jthickens — APPROVED** [review]

**Review by jthickens — CHANGES_REQUESTED** [review]
see comments

**Review by jthickens — APPROVED** [review]
lgtm - python should be reviewed by @AdnoC

**Review by AdnoC — APPROVED** [review]

**Pull request merged**